packages/@tailwindcss-upgrade/src/codemods/template/migrate-arbitrary-utilities.ts

```diff
@@ -159,14 +159,21 @@ export function migrateArbitraryUtilities(
         candidate.kind === 'arbitrary' ? candidate.value : (candidate.value?.value ?? null)
       if (value === null) return
 
-      let spacingMultiplier = spacing.get(designSystem)?.get(value)
+      let spacingMultiplier = spacing.get(designSystem)?.get(value) ?? null
+      let rootPrefix = ''
+      if (spacingMultiplier !== null && spacingMultiplier < 0) {
+        rootPrefix = '-'
+        spacingMultiplier = Math.abs(spacingMultiplier)
```

Comment:
This is also an option 🤔
```suggestion
        spacingMultiplier *= -1
```